app/views/training/pages/_citation.html.slim

```diff
@@ -0,0 +1,8 @@
+.hf-citation
+  .gem-c-metadata data-module="gem-toggle metadata" data-gem-toggle-module-started="true" data-metadata-module-started="true"
+    dl.gem-c-metadata__list data-module="gem-track-click" data-gem-track-click-module-started="true"
+      dt.gem-c-metadata__term Published:
+      dd.gem-c-metadata__definition = format_date(content.created_at.to_fs)
```

Comment:
@instantrick @martikat we decided it best to use the date of first publication of the module not the page as recorded by the `ModuleRelease`. This needs changing.